### ISSUE TITLE
Fix detectable accessibility violations on /advantage

### ISSUE THREAD

**Issue by sowasred2012** [canonical/ubuntu.com#10578]
Running the Cypress accessibility tests reveals a few issues:

![image.png](https://images.zenhubusercontent.com/5cd93aaf2406d87914830598/f3b6ac13-21f8-48f5-96cd-e923e17fb492)

**Comment by carkod** [comment]
Advantage no longer exists
